libs/auth/package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "build": "tsc --build tsconfig.build.json",
     "clean": "rm -rf build *.tsbuildinfo",
+    "configure-java-card": "ts-node scripts/configure_java_card.ts",
```

Comment:
I've noticed other scripts use `esbuild-runner`. Might be something to look into for consistency

Reply:
Ah nice, good observation! Switched over: https://github.com/votingworks/vxsuite/pull/3069/commits/d66623467c4cc5081bb1785d07c04dc4736a31f2